tools/run_tests/python_utils/port_server.py

```diff
@@ -85,6 +96,7 @@ def can_bind(port, proto):
 def refill_pool(max_timeout, req):
   """Scan for ports not marked for being in use"""
   chk = list(range(1025, 32766))
+  chk = [p for p in chk if p not in cronet_restricted_ports]
```

Comment:
nit: one-character identifiers ("p") are against python codestyle
nit: you can just do `port for port in range(....) if ...` .